G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Api/SeleniumManager.cs

```diff
@@ -112,6 +112,25 @@ public static void QuitCurrentWrapper()
             }
         }
 
+        public static void CleanUp()
+        {
+            var tempFolder = Path.GetTempPath();
+            var tempScopedFolders = Directory.GetDirectories(tempFolder, "scoped_dir*", SearchOption.TopDirectoryOnly);
+            foreach (var scopedFolder in tempScopedFolders)
+            {
+                try
+                {
+                    var directory = new DirectoryInfo(scopedFolder);
+                    directory.Delete(true);
```

Comment:
A static `Directory.Delete()` would be shorter. (Un)fortunately I didn't make it in time with this remark ;-)